content/docs/learning-guide/tutorials/entities-and-prefabs/entity-and-prefab-basics.md

```diff
@@ -0,0 +1,216 @@
+
+1. Select the entities and prefab instances you want included in the new prefab. You can select multiple entities and prefab instances by holding **SHIFT** and clicking on each entity and prefab instance. Alternatively, you can **drag** a selection box around a group of entities and prefab instances in Entity Outliner or in the viewport.
+1. In Entity Outliner, **right-click** one of the selected entities or prefab instances and choose **Create Prefab...** from the context menu.
+1. In the **Save As…** window, navigate to your project’s `Prefabs` directory, supply a name for the prefab, and choose **Save**.
+
+In the following video, a prefab instance and an entity with a hierarchy are selected for a new prefab. When the new prefab instance is opened in [Focus Mode](#edit-a-prefab), it displays that the hierarchy is maintained in the new prefab.
+
+{{< video src="/images/learning-guide/tutorials/entities-and-prefabs/multiple-entity-prefab.mp4" info="Creating a prefab from a collection of entities and prefabs." autoplay="true" loop="true" width="450" >}}
+
+## Instantiate a prefab
+
+Prefabs can be added to a level or another prefab as a prefab instance. To instantiate a prefab, do the following:
+
+1. **Right-click** in Entity Outliner or the viewport and choose **Instantiate Prefab...** from the context menu.
+1. In the **Pick Prefab** window, navigate to the prefab file you want to instantiate and select it. Choose **OK**.
+
+![Instantiate a prefab in Entity Outliner.](/images/learning-guide/tutorials/entities-and-prefabs/instantiate-prefab.png)
+
+Alternatively, you can **drag** a `.prefab` from Asset Browser into Entity Outliner or the viewport to instantiate it, as demonstrated in the following video.
+
+{{< video src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-instance.mp4" info="Drag and drop to instantiate a prefab." autoplay="true" loop="true" width="900" >}}
+
+## Naming entities and prefab instances
+
+The name of a prefab instance is derived from the prefab file name. However, each prefab instance can have a unique name. You can edit the name of a selected  entity or prefab instance with any of the following methods:
+
+* **Click** the name of the entity or prefab instance in Entity Outliner to edit it.
+* Press **F2** to edit the name.
+* In Entity Outliner, **right-click** on the entity or prefab instance and choose **Rename** from the context menu.
+* In **Entity Inspector**, edit the name field.
+
+![Renaming a prefab.](/images/learning-guide/tutorials/entities-and-prefabs/rename-prefab.png)
+
+{{< note >}}
+Entities and prefab instances aren't required to have unique names. When you instantiate a prefab, or duplicate an entity or a prefab instance, the name of the instance or duplicate is the same as the source. Each entity and prefab instance has a unique Entity ID that is used by O3DE to identify them. You can view an entity or prefab instance Entity ID in Entity Inspector.
```

Comment:
@dm-amzn , I believe that the entity id of the container entity of a prefab instance is not visible in the inspector. Can you check once and correct this paragraph?